pages/dkp/kommander/2.2/install/mgmt-cluster-apps/index.md

```diff
@@ -0,0 +1,50 @@
+menuWeight: 40
+excerpt: Management cluster application minimum resources and storage requirements
+beta: false
+enterprise: true
+---
+
+This section details requirements for management cluster-specific applications. For the list of all available platform applications, see [Platform Application Configuration Requirements](../../../projects/applications/platform-applications/application-requirements/).
+
+This table describes the workspace platform applications specific to the management cluster, minimum resource requirements, and minimum persistent storage requirements.
+
+| Name                    | Minimum Resources Suggested  | Minimum Persistent Storage Required                       | Deployed by Default |
+|-------------------------|------------------------------|-----------------------------------------------------------|---------------------|
+| cert-manager            | cpu: 10m<br>memory: 32Mi     |                                                           | Yes                 |
+| reloader                | cpu: 100m<br>memory: 128Mi   |                                                           | Yes                 |
+| traefik                 | cpu: 500m                    |                                                           | Yes                 |
+| kubecost                | cpu: 700m<br>memory: 1700Mi  | # of PVs: 3<br>PV sizes: 0.2Gi, 2Gi, 32Gi (total: 34.2Gi) | Yes                 |
+| kube-prometheus-stack   | cpu: 300m<br>memory: 1500Mi  | # of PVs: 1<br>PV sizes: 50Gi                             | Yes                 |
+| kube-oidc-proxy         |                              |                                                           |                     |
+| prometheusadapter       | cpu: 1000m<br>memory: 1000Mi |                                                           | Yes                 |
+| kubernetes-dashboard    |                              |                                                           | Yes                 |
+| nvidia                  |                              |                                                           | No                  |
+| logging-operator        |                              |                                                           | No                  |
+| grafana-loki            |                              |                                                           | No                  |
+| grafana-logging         |                              |                                                           | No                  |
+| fluentbit               | cpu: 350m<br>memory: 350Mi   |                                                           | No                  |
+| minio-operator          |                              |                                                           | No                  |
+| gatekeeper              |                              |                                                           | No                  |
```

Comment:
I am pretty sure these are all installed by default on the Management cluster.   They are not installed on the 'bootstrap' cluster but that's not what we're talking about here...